readme.md

```diff
@@ -1 +1,19 @@
-# Spring Boot application
\ No newline at end of file
+# Spring Boot application
+
+## Import Database
+
+1. Log in to the database as root or another user with sufficient privileges to create new databases.
+ ```mysql -u root -p```
+2. Create a new database called ```springbootapp```.
+    ```
+    CREATE DATABASE springbootapp;
+    ```
+3. * Run the script located in ``src/db/springbootapp_db_script.sql``
+   
+   OR
+   
+   * Through cmd, enter the following
+   
+   ```
+   mysql -u username -p springbootapp < /path/to/springbootapp_db_script.sql
+   ```
```

Comment:
IntelliJ maybe does not come with a markdown validator though you can use that of VSCode or [IntelliJ Markdown Plugin](https://plugins.jetbrains.com/plugin/7793-markdown-support)

The markdown has several issues
1. Single line code snips use one backtick not three
1. Code block with three backticks must specify the language like ```bash
1. Listing numbers can be done via an ordered list using `1.`
1. No need to indent code blocks
1. Use blank lines between code blocks

These would be normally caught in a Markdown linter either on VScode or some intellij plugin.